src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -798,7 +798,7 @@ public static IntPtr GetHINSTANCE(Module m)
                     return new System.NotImplementedException();
                 //case HResults.E_POINTER:
                 case HResults.RO_E_CLOSED:
```

Comment:
```suggestion
                case HResults.COR_E_OBJECTDISPOSED:
                case HResults.RO_E_CLOSED:
```
We can share the implementation for both cases.